charts/tidb-cluster/templates/scripts/_start_restore.sh.tpl

```diff
@@ -0,0 +1,23 @@
+set -euo pipefail
+
+dirname=`date +%Y-%m-%dT%H%M%S`-${MY_POD_NAME}
+dataDir=/data/${dirname}
+mkdir -p ${dataDir}/
+host=`echo {{ .Values.clusterName }}_TIDB_SERVICE_HOST | tr '[a-z]' '[A-Z]'`
+
+{{- if .Values.restore.gcp }}
+downloader \
```

Comment:
This fails for me. `/bin/sh: line 7: downloader: command not found`

Reply:
The latest image is just pushed to DockerHub.